### PR TITLE
fix(console): use connector dark logo

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

use `logoDark` for all possible places.

## Testing

截屏2022-12-20 下午5 01 11

截屏2022-12-20 下午5 03 32

截屏2022-12-20 下午5 04 14

### CODE REVIEW

**Review by gao-sun — APPROVED** [review]
decent

**Review by xiaoyijun — APPROVED** [review]
LGTM

File: packages/console/src/components/ConnectorLogo/index.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import type { ConnectorResponse } from '@logto/schemas';
+import { AppearanceMode } from '@logto/schemas';
+
+import { useTheme } from '@/hooks/use-theme';
+
+type Props = {
+  className?: string;
+  connector: Pick<ConnectorResponse, 'logo' | 'logoDark'>;
```
Comment (charIeszhao) [inline packages/console/src/components/ConnectorLogo/index.tsx:8]: nit: This is no longer a connector object, consider renaming it to `data`. Or simply just split it into two properties: logo and darkLogo

**Review by charIeszhao — APPROVED** [review]

**Review by darcyYe — APPROVED** [review]

File: packages/console/src/components/ConnectorLogo/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/components/ConnectorLogo/index.tsx]: renamed

**Pull request merged**